docs/topics/native/native-definition-file.md

```diff
@@ -74,7 +74,6 @@ For more information, see the corresponding sections below.
 | [`compilerOpts`](#pass-compiler-and-linker-options)                                 | Compiler options that the cinterop tool passes to the C compiler.                                                                                                                                                        |
 | [`linkerOpts`](#pass-compiler-and-linker-options)                                   | Linker options that the cinterop tool passes to the linker.                                                                                                                                                              |
 | [`excludedFunctions`](#ignore-specific-functions)                                   | A space-separated list of function names that should be ignored.                                                                                                                                                         |                                              
-| `excludedMacros`                                                                    |                                                                                                                                                                                                                          |
```

Comment:
JFYI
looks like this property is commented out in line 91
so I think it should be deleted here